src/store/manage-songs/manageSongs.thunk.js

```diff
@@ -0,0 +1,10 @@
+import { auth } from '../../firebase'
+import { getSongsByUser } from './manageSongs.provider'
+import { setSongs } from './manageSongs.slice'
+
+export const handleGetSongs = () => async (dispatch) => {
+  const user = auth.currentUser
```

Comment:
since the auth ref is an instance of firebase, the user prop can live inside of the firebase provider, not in the thunk, also omit to pass the uid value as an argument